Tests/IntegrationTests/Card Tests/Validators/CardHolderNameValidatorTests.swift

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright (c) 2024 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) @testable import Adyen
+@testable @_spi(AdyenInternal) import AdyenCard
+import XCTest
+
+final class CardHolderNameValidatorTests: XCTestCase {
+
+    func testValidNames() {
+        let validator = CardHolderNameValidator()
+        
+        XCTAssertTrue(validator.isValid("abc"))
+        XCTAssertTrue(validator.isValid("test name 123 ,.;[]'."))
```

Comment:
This is very interesting rule for the name validation :) Are there any characters that are not allowed?

Reply:
Apparently not :)